models/ai85net-actiontcn.py

```diff
@@ -114,13 +114,13 @@ def forward(self, x):
         """Forward prop"""
         batch_size = x.shape[0]
         num_frames = x.shape[1]
-        cnnoutputs = torch.zeros(batch_size, num_frames, self.cnn_out_channel,
-                                 self.cnn_out_shape[0], self.cnn_out_shape[1],
-                                 device=x.get_device())
-        for i in range(num_frames):
-            prep_out = self.create_prep(x[:, i])
-            cnnoutputs[:, i] = self.create_cnn(prep_out)
+        cnnoutputs = torch.zeros_like(x)
+        cnnoutputs = cnnoutputs[:, :, :self.cnn_out_channel, :self.cnn_out_shape[0],
+                                :self.cnn_out_shape[1]]
 
+        for i in range(15):
```

Comment:
Is the number 15 true? Shouldn't it be a parameter depending on the size of x?

Reply:
This was realized due to another limitation of symbolic trace. When symbolic trace is realized, parameters from x.shape become proxy objects and when they are used with range function an error is thrown as "TypeError: 'Proxy' object cannot be interpreted as an integer". 

For clarity, in my last commit, I moved num_frames to the model class as a parameter that doesn't depend on x.shape and changed this hardcoded 15 with self.num_frames.